Shared/src/com/pacoapp/paco/shared/util/ExperimentHelper.java

```diff
@@ -236,8 +236,11 @@ public static boolean isAnyGroupOngoingDuration(ExperimentDAO experiment) {
     List<ExperimentGroup> listeningExperimentGroups  = new ArrayList();
     List<ExperimentGroup> experimentGroups = experiment.getGroups();
     for (ExperimentGroup experimentGroup : experimentGroups) {
-      if (experimentGroup.getBackgroundListen() && experimentGroup.getBackgroundListenSourceIdentifier().equals(sourceIdentifier)) {
-        listeningExperimentGroups.add(experimentGroup);
+      if (experimentGroup.getBackgroundListen()) {
```

Comment:
Hi Bob, Thanks for your comment!

I am actually using backgroundListen, because this seemed like the way to go to get any data passed by the `Bundle extras` into the datastore. I realize that this functionality is meant to be used by external apps trying to send information to Paco, but it seemed to be applicable to internal triggers trying to pass around data too.

I noticed there was no clear path on how to do this in Paco just yet (e.g. the `duration` field in line 77 of `BroadcastTriggerService.java` doesn't seem to get stored yet. Do you think it would make more sense to use a different method, similar to `persistAccessibilityData()` to store the data into the database? For example, a `persistInstallationData()` that would get called for every experiment having an app install or remove trigger?

Would love to hear what you think!